projects/plugins/jetpack/_inc/client/newsletter/email-settings.jsx

```diff
@@ -202,11 +214,7 @@ const EmailSettings = props => {
 					disabled={ featuredImageInputDisabled }
 					checked={ isFeaturedImageInEmailEnabled && isSubscriptionsActive }
```

Comment:
Would it be better if it was disabled in the OFF position? Maybe we could force it to be "unchecked" if featuredImageInputDisabled.

Reply:
That makes sense. I updated it in 60b49651fa46705b378e766dc17a9214c1563d4b